product_multicompany_submit/tests/test_multicompany_submit.py

```diff
@@ -0,0 +1,165 @@
+#  Copyright (c) Akretion 2020
+#  License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
+
+from odoo.tests import common
+from odoo.exceptions import AccessError
+
+
+class TestMulticompanySubmit(common.TransactionCase):
+    def setUp(self):
+        super(TestMulticompanySubmit, self).setUp()
+        # active multicompany
+        self.env.ref("product.product_comp_rule").active = True
+
+        self.company_approver = self.env["res.company"].create(
+            {"name": "company of approver"}
+        )
+        self.company_submitter = self.env["res.company"].create(
+            {"name": "company of submitter"}
+        )
+        self.company_other = self.env["res.company"].create(
+            {"name": "company of other user"}
+        )
+
+        self.user_approver = self.env["res.users"].create(
```

Comment:
Can you create this 3 users in demo data, so I will be easier to test and make demo?